wgengine/router/router.go

```diff
@@ -72,6 +72,49 @@ type Config struct {
 	NetfilterMode    preftype.NetfilterMode // how much to manage netfilter rules
 }
 
+func (a *Config) Equal(b *Config) bool {
+	if len(a.LocalAddrs) != len(b.LocalAddrs) {
+		return false
+	}
+	if len(a.Routes) != len(b.Routes) {
+		return false
+	}
+	if len(a.LocalRoutes) != len(b.LocalRoutes) {
+		return false
+	}
+	if len(a.SubnetRoutes) != len(b.SubnetRoutes) {
+		return false
+	}
+	if a.SNATSubnetRoutes != b.SNATSubnetRoutes {
+		return false
+	}
+	if a.NetfilterMode != b.NetfilterMode {
+		return false
+	}
+
+	for i := range a.LocalAddrs {
+		if a.LocalAddrs[i] != b.LocalAddrs[i] {
```

Comment:
Wasn't sure if the "can compare with ==" comment in netaddr is only true for `IP`, or if we can also use it for `IPPrefix`

Reply:
also IPPrefix and IPPort and IPRange. I guess docs could be more clear.